.github/workflows/package_deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: Package and Deploy
+
+on:
+  push:
+    branches:
+      - feature/package-artifact
+  workflow_dispatch:
+
+
+jobs:
+  build_package_deploy:
+    name: Deploy Module
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup PowerShell module cache
+        id: cacher
+        uses: actions/cache@v3
+        with:
+          path: "~/.local/share/powershell/Modules"
+          key: ${{ runner.os }}-PSModules
+      - name: Setup
+        shell: pwsh
+        run: |
+          ./Tools/setup.ps1
+          Invoke-Build -Task ShowInfo
+      - name: Build
+        shell: pwsh
+        run: |
+          Invoke-Build -Task Clean, Build
+      - name: Package
+        shell: pwsh
+        run: |
+          Invoke-Build -Task Package
```

Comment:
can't we inline this with the previous task?

```pwsh
Invoke-Build -Task Clean, Build, Package
```

it's not like the tasks would benefit from the split, right?

Reply:
Keeping the steps separate means it's easy to see quickly where it failed and why.

Not benefit to merging them together IMO. But you're call.